### PR TITLE
Add three missing layouts

### PR DESCRIPTION
**Pull request by fauxpark**
## Description

And adding layout support for a couple of applicable boards :)

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [x] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by zvecr** [comment]
@noroadsleft @yanfali wont this need info.json updates and what about qmk_configurator default keymap updates due to the LAYOUT alias?

**Comment by fauxpark** [comment]
The Canoe has no default keymap in Configurator yet. Will update the Tragicforce68 default though.

File: layouts/default/60_iso_split_bs_rshift/default_60_iso_split_bs_rshift/keymap.c [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+     * │ Tab │ Q │ W │ E │ R │ T │ Y │ U │ I │ O │ P │ [ │ ] │     │
+     * ├─────┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┐ Ent│
+     * │ Caps │ A │ S │ D │ F │ G │ H │ J │ K │ L │ ; │ ' │ # │    │
+     * ├────┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴───┴┬───┤
+     * │Shft│ \ │ Z │ X │ C │ V │ B │ N │ M │ , │ . │ / │Shift │Sft│
+     * ├────┼───┴┬──┴─┬─┴───┴───┴───┴───┴───┴──┬┴───┼───┴┬────┬┴───┤
+     * │Ctrl│GUI │Alt │                        │ Alt│ GUI│Menu│Ctrl│
+     * └────┴────┴────┴────────────────────────┴────┴────┴────┴────┘
+     */
+    [0] = LAYOUT_60_ansi_split_bs_rshift(
```
Comment (noroadsleft) [inline layouts/default/60_iso_split_bs_rshift/default_60_iso_split_bs_rshift/keymap.c]:
```suggestion
    [0] = LAYOUT_60_iso_split_bs_rshift(
```

File: layouts/default/68_iso/default_68_iso/keymap.c [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+     * │ Tab │ Q │ W │ E │ R │ T │ Y │ U │ I │ O │ P │ [ │ ] │     ││Del│PgD│
+     * ├─────┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┐ Ent│└───┴───┘
+     * │ Caps │ A │ S │ D │ F │ G │ H │ J │ K │ L │ ; │ ' │ # │    │
+     * ├────┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴───┴────┤┌───┐
+     * │Shft│ \ │ Z │ X │ C │ V │ B │ N │ M │ , │ . │ / │ Shift    ││ ↑ │
+     * ├────┼───┴┬──┴─┬─┴───┴───┴───┴───┴───┴──┬┴───┼───┴┬────┬─┬──┴┼───┼───┐
+     * │Ctrl│GUI │Alt │                        │ Alt│ GUI│Ctrl│ │ ← │ ↓ │ → │
+     * └────┴────┴────┴────────────────────────┴────┴────┴────┘ └───┴───┴───┘
+     */
+    [0] = LAYOUT_68_ansi(
```
Comment (noroadsleft) [inline layouts/default/68_iso/default_68_iso/keymap.c]:
```suggestion
    [0] = LAYOUT_68_iso(
```

File: layouts/default/68_iso/readme.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+# 68_iso
+
+    LAYOUT_68_iso
+
+This is the 68 (actually 69) key ISO layout made popular by boards such as the Magicforce 68 and Varmilo VA68M.
```
Comment (noroadsleft) [inline layouts/default/68_iso/readme.md]: If we want to get technical:

```suggestion
This is the 68 (actually 69) key ISO layout made popular by boards such as the Magicforce 69-key and Varmilo VA69M.
```

I don't actually know what the ISO Magicforce 68 is called officially, but Varmilo's board is the VA69M.

**Comment by noroadsleft** [comment]
> The Canoe has no default keymap in Configurator yet. Will update the Tragicforce68 default though.

Tragicforce default on Configurator actually won't need updating at all. The LAYOUT -> LAYOUT_68_ansi alias will take care of that and it'll still work as intended.

info.json updates all LGTM.

File: layouts/default/60_iso_split_bs_rshift/default_60_iso_split_bs_rshift/keymap.c (hunk 1 above)
Comment (fauxpark) [reply layouts/default/60_iso_split_bs_rshift/default_60_iso_split_bs_rshift/keymap.c]: 🤦‍♂ should have gone looking for boards with this layout so Travis would have caught it too.

**Pull request merged**